ui/components/resolvers/Security.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
It's ok to have this component in the repo but we whould not display it anywhere yet.